README.md

```diff
@@ -12,33 +12,98 @@ They can also be used independently of Witan as a standalone demographic models
+```
+A GSS code must always be specified to tell the model which geographic area the projections are for. See the GSS codes that are accepted [here](#gss-code).
 
 ## Options
 
-FIXME: listing of options this app accepts.
+The projections can be customised by adjusting the options below. Read more about how the .jar file works and what input data is expected in the [docs](https://github.com/MastodonC/witan.models.demography/tree/master/doc/run-model.md).
+
+* [`-c` or `--gss-code`](#gss-code)
+* [`-i` or `--input-config`](#input-config)
+* [`-o` or `--output-projections`](#output-projections)
+
+
+The last two options have default values if not specified:
+
+* default input configuration filepath: "default_config.edn"
+* default output projection filepath: "ccm_projections.csv"
+
+These default settings mean the "default_config.edn" file is located inside this repository and the output will be created as "ccm_projections.csv" inside this repository as well.
+
+The previous command can thus been run as follows:
+
+```
+	$ java -jar witan.models.demography/target/uberjar/witan.models.demography-0.1.0-SNAPSHOT-standalone.jar -c "E06000023"
+```
+
+Example:
+
+To save the projections output to a different directory, with a different filename:
+
+```
+	$ java -jar witan.models.demography/target/uberjar/witan.models.demography-0.1.0-SNAPSHOT-standalone.jar -c "E06000023" -o /home/user/Documents/ccm_projections/Bristol_projections.csv" 
+```
+
+### gss-code
+
+The geographical area for the projection must be specified. This is done with the 9-digit GSS code (Government Statistical Service code), or name, of one of the following:
+
+  * an English unitary authority (starts with "E06")   
+  * an English non-metropolitan district (starts with "E07")   
+  * an English metropolitan borough (starts with "E08")   
+  * a London borough (starts with "E09")   
+  * Northern Ireland, Scotland, or Wales (country codes "N92000002", "S92000003", or "W92000004"; smaller geographies not yet available)
+
+### input-config
+
+The configuration file contains file paths to the model input datasets and model parameters that can be adjusted by the user. The parameters in this file are described [here](doc/run-model.md).
 
-## Examples
+The [default config file](https://github.com/MastodonC/witan.models.demography/blob/master/default_config.edn) should be used as a template and edited at its current location or copied (and modified) anywhere else on your machine.
 
+### output-projections
 
-### Bugs
+This is the path to the file where the final projection should be saved. The projections are returned in one csv file. This file contains the historical population data used in the projection, with the projected population appended. The output path can be anywhere on your machine as long as the directories on the path exist. 
```

Comment:
I'll add the column headers for the output to run-models.md, which also has the column headers for the input data.